newIDE/app/src/ResourcesList/LocalResourceSources.js

```diff
@@ -200,14 +226,27 @@ const selectLocalResourcePath = (
     title: string,
     name: string,
     extensions: Array<string>,
-  }
+  },
+  loadLatestPath: (project: gdProject, kind: ResourceKind) => string,
+  saveLatestPath: (
+    project: gdProject,
+    kind: ResourceKind,
+    path: string
+  ) => void,
+  kind: ResourceKind
 ): Promise<Array<string>> => {
   return new Promise((resolve, reject) => {
     if (!dialog) return reject('Not supported');
 
     const properties = ['openFile'];
     if (options.multiSelections) properties.push('multiSelections');
-    const projectPath = path.dirname(project.getProjectFile());
+    let projectPath = path.dirname(project.getProjectFile());
+
+    // Load latestPath and update projectPath if not undefined
+    const latestPath = loadLatestPath(project, kind);
```

Comment:
Don't add a comment for this, because you can rewrite a line like this in a way it's almost like English:

```suggestion
    const latestPath = getLastUsedPath(project, kind) || projectPath;
```

"The latest path is the last used path, for this project and kind, OR if not defined, it's the project path".

Clear and concise :)